Bigtable/src/ReadModifyWriteRowRules.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\V2\ReadModifyWriteRule;
+
+/**
+ * Represents collection of ReadModifyWriteRule to perform operation on Bigtable table.
+ * This is used in readModifyWriteRow operation on row in Bigtable table.
+ */
+class ReadModifyWriteRowRules
+{
+    /**
+     * @var array
+     */
+    private $options;
+
+    /**
+     * @var array ReadModifyWriteRule
+     */
+    private $rules = [];
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    /**
+     * Appends the value to the existing value of the cell. If targeted cell is unset,
+     * it will be treated as containing the empty string.
+     *
+     * @param string $familyName Family name of the row.
+     * @param string $qualifier Column qualifier of the row.
+     * @param string $value Value of the Column qualifier.
+     *
+     * @return ReadModifyWriteRowRules returns current ReadModifyWriteRowRules object.
```

Comment:
The description here doesn't provide a lot of value, since we can already tell it from the type. Can we please omit? (applies to the other methods)

Reply:
Description says its returning current object. With just type it might mean that it may be returning new object.

Reply:
That's fair, perhaps we can notate this class as a builder in the class description then instead?